v7/recent_posts_json/recent_posts_json.py

```diff
@@ -131,7 +132,10 @@ def make_json(self, posts, descriptions, previewimage, output_path, lang):
             recent_posts.append(entry)
         data = json.dumps(recent_posts, indent=2, sort_keys=True)
         with io.open(output_path, "w+", encoding="utf8") as outf:
-            outf.write(data)
+            if sys.version_info[0] != 3:
```

Comment:
Absolutley we can do it like that. Don't know why the first commit is wrong though, isn't json utf-8 by design? In ptyhon2 ".decode('utf-8')" works on both string and unicode objects